internal/data/evtypes/models/event_type.go

```diff
@@ -56,8 +56,9 @@ func (e EventType) Flag() string {
 	}
 }
 
-func (e EventType) Resource() resources.EventStaticMeta {
-	return resources.EventStaticMeta{
+func (e EventType) Resource(qr bool) resources.EventStaticMeta {
```

Comment:
not `qr` it's `isAdmin` or `revealQR` etc. semantics; or let's even use it outside of the function, in admin endpoint, like:
```go
r := evType.Resource()
r.QrCodeValue = evType.QRCodeValue
```